docker-compose.yml

```
@@ -0,0 +1,36 @@
+services:
+  mysqldb:
+    platform: linux/arm64
+    image: mysql:latest
+    restart: unless-stopped
+    env_file: ./.env
+    environment:
+      - MYSQL_USER=$MYSQLDB_USER
+      - MYSQL_PASSWORD=$MYSQLDB_PASSWORD
+      - MYSQL_ROOT_PASSWORD=$MYSQLDB_ROOT_PASSWORD
+      - MYSQL_DATABASE=$MYSQLDB_DATABASE
+    ports:
+      - "$MYSQLDB_LOCAL_PORT:$MYSQLDB_DOCKER_PORT"
+    healthcheck:
+      test: [ "CMD", "mysqladmin", "ping", "-h", "localhost" ]
+      interval: 30s
+      timeout: 30s
+      retries: 3
+  app:
+    depends_on:
+      - mysqldb
+    restart: on-failure
+    image: book-service
+    build: .
+    env_file: ./.env
+    ports:
+      - "$SPRING_LOCAL_PORT:$SPRING_DOCKER_PORT"
+      - "$DEBUG_PORT:$DEBUG_PORT"
+    environment:
+      SPRING_APPLICATION_JSON: '{
+         "spring.datasource.url"  : "jdbc:mysql://localhost:3306/spring",
```

```suggestion
        "spring.datasource.url"  : "jdbc:mysql://localhost:3306/spring",
```
Use variadles instead of port and schema name